var/spack/repos/builtin/packages/seacas/package.py

```diff
@@ -47,9 +48,9 @@ class Seacas(CMakePackage):
     variant('applications', default=True,
             description='Build all "current" SEACAS applications. This'
             ' includes a debatable list of essential applications: '
-            'aprepro, conjoin, ejoin, epu, exo2mat, mat2exo, '
-            'exo_format, exodiff, explore, grepos, '
-            'nemslice, nemspread')
+            'aprepro, conjoin, cpup, ejoin, epu, exo2mat, mat2exo, '
+            'exo_format, exodiff, explore, grepos, io_*, '
```

Comment:
Can you avoid `*` in variant names?

Reply:
That isn't a variant, but I will remove the `*` just to avoid any confusion.  It was meant to be a shortcut for several applications whose names all start with `io_`.

Reply:
ah yes, sorry, my mistake, carry on...